waterloop-site/src/components/SustainableTech/FAQ/collapsible.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import styled from 'styled-components';
+import {ArrowIosDownwardOutline, ArrowIosUpwardOutline} from '@styled-icons/evaicons-outline';
```

Comment:
Rather than have an up arrow and down arrow as individual components, would be able to have just the one down arrow component and when the user expands the dropdown, the arrow flips 180 degrees (with a transition to make the rotation look smooth)?
Although this is a small detail, it would be awesome if you could do it.